docs/application/vscode-ext/Tizen/dotnet.md

```diff
@@ -1,4 +1,4 @@
-# .NET application development
+# Installing Visual Studio Code Extension For Tizen
```

Comment:
Thanks for the update work for this page usage.
However while sharing the stg link to other members,
I found out that I missed checking the links related with the page.
Some links need to be updated due to the changes in the contents.

I grepped "dotnet.md" string in the document,
and among the results, below seems to need an update.

Would you please check the below suggestions, test in your local, and share your opinion?
If you agree the suggestion please apply it.

- https://docs1.stg.tizen.org/staging/1627/application/dotnet/get-started
   `If you are using Linux, you can use [Visual Studio Code Extension](../../vscode-ext/Tizen/dotnet.md) to create your first app.`
   According to the sentence, it seems better to use the `first-app.md`
   
- https://docs1.stg.tizen.org/staging/1627/application/vscode-ext/Tizen/hybrid/
   `To debug dotnet application, set dotnet app as main app, set native app as dependency app to dotnet app. Refer to [Debug your application in emulator](dotnet.md) section for details.`
   Contextually, how about linking to `../../dotnet/get-started/first-app.md#debug-your-application-in-emulator`?

- https://docs1.stg.tizen.org/staging/1627/application/vstools/Tizen/hybrid/
   `To create a Tizen .NET project: Refer to [Create Tizen .NET project](dotnet.md)`
   Contextually, how about linking to `../../dotnet/get-started/first-app.md`?

Reply:
@mijin85cho Thank you for highlighting that! I have checked your suggestions on my local machine and they seem to be correct. So I have pushed the changes to this PR. Do check if it is okay.